libbpf-rs/src/object.rs

```diff
@@ -329,7 +327,9 @@ impl Object {
 
```

Comment:
as an aside, bpf_map__def() is deprecated, we should try to remove it before libbpf 1.0 release